docs/serving/autoscaling/concurrency.md

```diff
@@ -74,17 +74,20 @@ spec:
 
 ### Hard limit
 
-The hard limit has no global setting and can only be specified [per revision](./autoscaling-concepts.md).
-This particular per-revision setting is not an annotation; it is present on the revision's spec itself as `containerConcurrency`.
+The hard limit is always specified [per revision](./autoscaling-concepts.md) via the `containerConcurrency` field on the revision spec. This particular per-revision setting is not an annotation; it is present on the revision's spec itself as `containerConcurrency`.
+
+There is no global setting for the hard limit in the autoscaling config map (because `containerConcurrency` has implications outside of autoscaling, such as on buffering and queuing of requests), but a default value can be set for the Revision's `containerConcurrency` field in `config-defaults.yaml`.
```

Comment:
```suggestion
There is no global setting for the hard limit in the autoscaling ConfigMap, because `containerConcurrency` has implications outside of autoscaling, such as on buffering and queuing of requests. However, a default value can be set for the Revision's `containerConcurrency` field in `config-defaults.yaml`.
```